src/help/zaphelp/contents/cmdline.html

```diff
@@ -9,6 +9,19 @@
 <BODY BGCOLOR="#ffffff">
 <H1>Command Line</H1>
 
+To run ZAP via the command line, you will need to locate the ZAP startup script.<br/>
+<strong>Windows:</strong><br/>
+<pre>C:\Program Files (x86)\OWASP\Zed Attack Proxy\zap.bat</pre>
+<strong>Mac:</strong><br/>
+<pre>/Applications/OWASP\ ZAP.app/Contents/Java/zap.sh</pre>
+<strong>Linux:</strong> <code>zap.sh</code> will be below the directory where ZAP was installed.
+<br/>

```

Comment:
Maybe add another line break? The following text seems to apply only to Linux entry.